Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM node:alpine
+FROM ghcr.io/littlehorse-enterprises/alpine-nginx-nodejs/nginx-nodejs:main AS runner
+ENV NODE_ENV=production
+
 WORKDIR /app
 
-COPY ./ui/.next/standalone ./
-COPY ./ui/.next/static ./ui/.next/static
-COPY ./node_modules ./node_modules
+COPY ./ui/.next/standalone/ui ./
+COPY ./ui/.next/standalone/node_modules ./node_modules
+COPY ./ui/.next/static ./.next/static
+
 COPY ./entrypoint.sh ./
 
-ENV NODE_ENV=production \
-    PORT=3000 \
-    HOSTNAME="0.0.0.0"
 EXPOSE 3000
+ENV PORT=3000
+ENV HOSTNAME="0.0.0.0"
+ENV NODE_ENV="production"
 
-ENTRYPOINT [ "./entrypoint.sh" ]
+ENTRYPOINT [ "./entrypoint.sh" ]
```

Comment:
This overwrites the base image entrypoint

Reply:
<img width="717" alt="image" src="https://github.com/user-attachments/assets/63210258-dbc2-4622-8f0e-8a030eddd243">
It does not overwrite it as its in the `/app` directory